perception/lidar_apollo_instance_segmentation/src/detector.cpp

```diff
@@ -182,9 +182,9 @@ bool LidarApolloInstanceSegmentation::detectDynamicObjects(
 
   // move down pointcloud z_offset in z axis
   for (std::size_t i = 0; i < output.feature_objects.size(); i++) {
-    sensor_msgs::msg::PointCloud2 transformed_cloud;
-    transformCloud(output.feature_objects.at(i).feature.cluster, transformed_cloud, -z_offset_);
-    output.feature_objects.at(i).feature.cluster = transformed_cloud;
+    sensor_msgs::msg::PointCloud2 updated_cloud;
+    transformCloud(output.feature_objects.at(i).feature.cluster, updated_cloud, -z_offset_);
+    output.feature_objects.at(i).feature.cluster = updated_cloud;
   }
```

Comment:
@kobayu858 
Could you use range-based for loop here too?

Reply:
@veqcc 
Correction. In the future, if I find similar for loop even if it is not directly related to the cppcheck warning, should I fix it?

Reply:
If it is not a complicated fix, yes please.

Reply:
I understand.